.changeset/stale-lions-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore-backend': patch
```

Comment:
this seems like it belongs in a different PR

Reply:
removed from this PR